users/models.py

```diff
@@ -3,10 +3,13 @@
 
 
 def upload_location(instance, filename):
-	return "{}/{}".format(instance.id, filename)
+    return "{}/{}".format(instance.id, filename)
 
 class Profile(models.Model):
```

Comment:
instead of creating a duplicate model for user, create a custom user model by extending the built-in user module.